src/import.rs

```diff
@@ -30,26 +33,27 @@ pub struct Import {
     #[clap(long, action)]
     pub audio_previews: bool,
 
-    ///Increases the CLI verbosity.
-    #[clap(short, long, action)]
-    pub verbose: bool,
+    ///Sets the CLI verbosity. Allowed values are 'error', 'warn', 'info', 'debug' and 'trace'.
+    #[clap(short, long, default_value = "error")]
+    pub verbose: String,
 }
 
 impl Import {
     //TODO old logic; has to be changed to match new commands
     pub fn import(&mut self) {
-        println!("Importing audio from SD card at {:?}", self.device);
+        init_logging(&self.verbose);
```

Comment:
Should we actually initiate logging in the main?

Reply:
but i think with the clap_verbosity_flag it should be possible, let me try that out